user/chrome.md

```diff
@@ -19,3 +20,18 @@ addons:
 ## Headless mode
 
 You can use Google Chrome in [headless mode](/user/gui-and-headless-browsers/#Using-the-Chrome-addon-in-the-headless-mode).
+
+## Sandboxing
+
+For security reasons, Google Chrome is unable to provide sandboxing when it is running in the
+[container-based environment](https://docs.travis-ci.com/user/reference/overview/#Virtualization-environments).
+
+To use Chrome in the container-based environment, pass `--no-sandbox` when invoking the `chrome` command.
+
+```yaml
+sudo: false
+addons:
+  chrome: stable
+script:
+  - chrome --no-sandbox
```

Comment:
Is this `script` content a realistic use case?

Reply:
I am afraid not, but I could not think of a more realistic one. 😬 Suggestions welcome!

Reply:
I recommend giving an example in the context of one of the example projects referenced by a Google blog post: https://github.com/ebidel/headless-karma-travis

This would mean that it's a combination of an example `.travis.yml` change and a change to one's `karma.conf.js`, and you could say something about doing similarly for capybara or whatever else.

Reply:
Here's the specific spot where the chrome runner flags are managed in the case of the headless-karma-travis example: https://github.com/ebidel/headless-karma-travis/blob/da2131fbe130c45d32ea3fa138d9d475baf81f7e/karma.conf.js#L16-L18